core_contracts/dividends/dividends.py

```diff
@@ -195,12 +216,17 @@ def __init__(self, db: IconScoreDatabase) -> None:
 
         self._distribution_activate = VarDB(self._DISTRIBUTION_ACTIVATE, db, value_type=bool)
```

Comment:
Need an activate method to set this to True callable only from governance.

Reply:
Already present in [L230](https://github.com/balancednetwork/balanced-contracts/blob/development/core_contracts/dividends/dividends.py#L230)